bridge/src/main/java/com/bridge/ipc/SocketClient.java

```diff
@@ -0,0 +1,68 @@
+package com.bridge.ipc;
+
+import com.bridge.core.exceptions.renderHandlerExceptions.RenderException;
+import com.bridge.renderHandler.render.RenderManager;
+import java.io.IOException;
+import java.net.StandardProtocolFamily;
+import java.net.UnixDomainSocketAddress;
+import java.nio.ByteBuffer;
+import java.nio.channels.SocketChannel;
+import java.nio.file.Path;
+
+/**
+ * This class provides a client-side connection for inter-process communication (IPC)
+ * using Unix domain sockets.
+ */
+public class SocketClient {
+    public static final Path NAMESPACE =
+            Path.of(System.getProperty("java.io.tmpdir"), "events-socket.sock");
```

Comment:
The path of the server-socket of Screen is `/tmp/socket_console` and should be the same.
This path I think is of input-intermediate socket

Reply:
I've updated it to the suggested path